research/picai/preprocessing/prepare_data.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 import SimpleITK as sitk
 
-from research.picai.preprocessing import (
+from research.picai.preprocessing.preprocessing import (
```

Comment:
preprocessing.preprocessing is a little awkward, but if `preprocessing.py` is the most logical name for the file, we can keep it 🙂

Reply:
Maybe you could just import everything from the preprocessing file into the preprocessing subpackage in the __init__.py file?

Reply:
Or alternatively `from .preprocessing impot ...`

Reply:
Both seem like solid suggestions, I might go with a rename of preprocessing dir to data_utils in case we need to add data related files that are not necessarily preprocessing (ie augmentation) in the future